data_diff/__main__.py

```diff
@@ -31,7 +35,7 @@ def parse_table_name(t):
 @click.option("-t", "--update-column", default=None, help="Name of updated_at/last_updated column")
```

Comment:
What do you think about exposing `--min-update` and `--max-update` here too while we're at it?

Reply:
Maybe? Let's leave that for another PR though. Always easy to add.